x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -15238,8 +15238,8 @@
     "xpack.securitySolution.auditd.errorFromDescription": "エラー：",
     "xpack.securitySolution.auditd.executedDescription": "実行",
     "xpack.securitySolution.auditd.executionOfForbiddenProgramDescription": "禁止されたプログラムの実行",
-    "xpack.securitySolution.auditd.failedLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
-    "xpack.securitySolution.auditd.inDescription": "in",
+    "xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
```

Comment:
I don't think this line needs to change. Looks like there is a typo now in the translation id.

Reply:
thanks for the second pair of eyes